worker/worker.go

```diff
@@ -25,6 +25,8 @@ import (
 	"go.sia.tech/renterd/api"
 	"go.sia.tech/renterd/build"
 	"go.sia.tech/renterd/config"
+	"go.sia.tech/renterd/internal/gouging"
+	clientV2 "go.sia.tech/renterd/internal/rhp/v2"
```

Comment:
`nit/question` I'm wondering why you went with `clientV2` as an alias here instead of `rhp2`, V2 is somewhat confusing in the context of the new txn types

Reply:
Mostly to avoid the confusion with `rhpv2` imports.